docker-compose.yaml

```diff
@@ -112,14 +120,25 @@ services:
       context: .
       dockerfile: ./docker/ui.Dockerfile
     container_name: rbac-ui
-    image: rbac-ui-production:${ISOLATION_ID-latest}
+    image: rbac-ui:${ISOLATION_ID-latest}
     ports:
       - "4200:4200"
     depends_on:
       - rbac-server
     environment:
       - HOST=${HOST:-localhost}
 
+  rbac-client:
```

Comment:
This is the change that @jbobo is referring to, and it is part of this PR along with the cleanup of the rbac-client Dockerfile. Both changes are improvements with regards to incorporating the new UI, but the missing dependency causes a dump of errors in the logs upon startup.